docs/user.adoc

```diff
@@ -52,7 +52,7 @@ While the Credentials API plugin provides a default _internal_ credentials store
 If you are using the _internal_ credentials store to store *high value* credentials then you will need to lock down your Jenkins configuration:
 
 * Apply all of the recommendations from the  https://wiki.jenkins.io/display/JENKINS/Securing+Jenkins[Securing Jenkins] wiki page.
-* No builds on the master (ideally zero executors)
+* No builds on the controller (ideally zero executors)
```

Comment:
Potentially wrong term

Reply:
What would you suggest?

Reply:
Still undefined due to lack of followup for the last 8 months.